Create get-started-with-playbooks.rst (#5461)
mattermost/docs: source/playbooks/get-started-with-playbooks.rst

Get started with Mattermost Playbooks
=====================================

> Access Mattermost playbooks from the Product Menu and select **Playbooks**.

Reviewer: Let's decide how we want to consistently introduce our product verticals:

The Channels onboarding page reads like this: "Access Mattermost Channels by selecting Channels from the Product list [icon-image] icon."

How do you feel about adopting that approach for the other 2 onboarding pages?

Author: I think that's reasonable. I've updated the docs.

Playbooks are workflows that can be repeated and refined over time. They can be applied to a range of situations and give stakeholders visibility into progress, process, and metrics. Playbooks are configurable and editable. You can add tasks to checklists, include automation behaviors, and provide insights into your processes in the form of metrics.

You can also choose a blank template and start from scratch - this is a good option if your use case is unique.

> Lastly, if you're unsure which option to choose, take a look at the **Learn how to use playbooks** template. This template breaks down the components of a playbook and you can also start a test run to see how everything fits together. If you're taking this option, you can stop reading here and enjoy the test run.

Reviewer: I wonder if formatting this as a tip may be valuable to new Playbooks users?

Tip: Take a look at the Learn to use playbooks template. This template breaks down the components...

> When you get started with Playbooks, it helps if you have a repeatable process in mind so you can apply what you know in order to get the most out of playbooks. For this document, we're going to refer to an incident response playbook and discuss the moving parts of its playbook.

Reviewer: ```suggestion
When you get started with Playbooks, it helps to have a repeatable process in mind up front. As a working example, we'll refer to an incident response playbook and discuss its moving parts.
```

Choose a template
-----------------

The first step is to choose the right template for your use case. There are pre-configured templates for specific scenarios. The checklists, actions, status updates, and retrospective settings for these templates may already be filled in and, where appropriate, enabled. You can always edit and adjust these settings - they're there to guide you - removing them doesn't negatively affect the playbook run.

In the incident response template, the template contains items that are relevant to incident resolution.

Keywords
--------

The most important part of a run is getting it started. An easy way to start a playbook run is to configure keywords. These keywords prompt a user to start the run when they're used. In the incident response playbook, the keywords are specific to critical incidents, for example ``sev-1`` and ``#incident``. It's unlikely that someone would use those terms in general conversation and, even if they do, they can elect not to start the playbook run when prompted.

Tasks and checklists
--------------------

> Tasks and checklists are the foundation of a template and a workflow. In an incident, it's critical to get stakeholders together as soon as possible so one of the first tasks is to add the on-call engineer to the channel, followed by starting a bridge call. When you're setting up these tasks you can add slash commands, at-mentions, and integrations with tools such as Zoom to make the initiation as seamless as possible.

Reviewer: ```suggestion
Tasks and checklists are the foundation of a template and a workflow. In an incident, it's critical to get stakeholders together as soon as possible, so one of the first tasks is to add the on-call engineer to the channel, followed by starting a bridge call. When you're setting up these tasks, you can add slash commands, at-mentions, and integrations with tools such as Zoom to make the initiation as seamless as possible.
```

Status updates
--------------

> Regular updates are important communication tools, especially in the middle of an outage. Channels can get very busy and overwhelming, and if you have more than one incident at a time it's often too noisy for stakeholders to keep track of everything.

Reviewer: ```suggestion
Regular updates are important communication tools, especially in the middle of an incident like an outage. Channels can get very busy and overwhelming, and if you have more than one incident at a time, it's often too noisy for stakeholders to keep track of everything.
```

> Use the **Broadcast update to other channels** to cut through the noise and share critical information with both channel members and other users in a dedicated channel.

Reviewer: ```suggestion
Use the **Broadcast update to other channels** option to cut through the noise and share critical information with both channel members and other users in a dedicated channel.
```

Get started with Mattermost Playbooks
=====================================

Access Mattermost Playbooks by selecting **Playbooks** from the **Product list** |product-list| icon.

.. |product-list| image:: ../images/products_E82F.svg
  :height: 24px
  :width: 24px

>   :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.

Reviewer 2: This page may need to be added to the TOC.
Also the icon is not rendering correctly on preview.

Author: Thanks, Amy!

I will add it to the TOC today and the icon isn't rendering because it's part of a PR from Carrie that'll be merged today - the icon should render then.

Author: So it's dependent on https://github.com/mattermost/docs/pull/5391 being merged first.

Playbooks are workflows that can be repeated and refined over time. They can be applied to a range of situations and give stakeholders visibility into progress, process, and metrics. Playbooks are configurable and editable. You can add tasks to checklists, include automation behaviors, and provide insights into your processes in the form of metrics.

When you get started with Playbooks, it helps to have a repeatable process in mind up front. As a working example, we'll refer to an incident response playbook and discuss its moving parts.

Visit the `Mattermost Playbooks documentation <https://docs.mattermost.com/playbooks/setting-up-playbooks.html>`_ to learn more.

Choose a template
-----------------

The first step is to choose the right template for your use case. There are pre-configured templates for specific scenarios. The checklists, actions, status updates, and retrospective settings for these templates may already be filled in and, where appropriate, enabled. You can always edit and adjust these settings - they're there to guide you - removing them doesn't negatively affect the playbook run.

.. tip:: 

>     If you're unsure which option to choose, take a look at the **Learn how to use playbooks** template. This template breaks down the components of a playbook and you can also start a test run to see how everything fits together. If you're taking this option, you can stop reading here and enjoy the test run. You can also choose a blank template and start from scratch - this is a good option if your use case is unique.

Author: ```suggestion
    Take a look at the **Learn how to use playbooks** template. This template breaks down the components of a playbook and you can also start a test run to see how everything fits together. If you're taking this option, you can stop reading here and enjoy the test run. You can also choose a blank template and start from scratch - this is a good option if your use case is unique.
```

> In the incident response template, the template contains items that are relevant to incident resolution. These are general items, to help you get started.

Reviewer 3: The first sentence here reads a little odd to me but 0/5. 

Also, there's possibly an extra comma in the second sentence?
`These are general items to help you get started.`
